packages/core/src/Stack/Stack.styles.ts

```diff
@@ -0,0 +1,28 @@
+import { css } from 'emotion';
+import tokens from '@contentful/f36-tokens';
+import { Spacing } from '../types';
+
+export const getStackStyles = ({
+  spacing,
+  flexDirection,
+}: {
+  spacing: Spacing;
+  flexDirection?: 'row' | 'column';
+}) => {
+  if (flexDirection === 'column') {
+    return css({
+      '> :not(style) ~ :not(style)': {
+        marginTop: tokens[spacing],
+        marginInline: 0,
+        marginBottom: 0,
+      },
+    });
+  }
+  return css({
+    '> :not(style) ~ :not(style)': {
```

Comment:
What is this selecting?

not style tags?

Reply:
got it!

if this is a lobotomized owl `* + *`
how should we call this `* ~ *`? it looks like a sad pokemon

Reply:
"defeated lobotomized pokemon"

Reply:
i think its a Wobbuffet.

Reply:
you are right!!!!

![Wobbuffet](https://user-images.githubusercontent.com/6597467/132513174-47d84efe-c5b5-46e4-ae86-c6737b30ea51.png)